### PR TITLE
Privatechat

### PR DESCRIPTION
**Pull request by sravanikarri**
fixing empty chat id

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/chat/component.jsx [hunk 1]
```diff
@@ -91,7 +91,7 @@ const Chat = (props) => {
                 hideLabel
                 onClick={() => {
                   actions.handleClosePrivateChat(chatID);
-                  Session.set('idChatOpen', 'public');
+                  Session.set('idChatOpen', '');
```
Comment (saiky76) [inline bigbluebutton-html5/imports/ui/components/chat/component.jsx:94]: When you did private chat did you make this change and now you are reverting?

Comment (sravanikarri) [reply bigbluebutton-html5/imports/ui/components/chat/component.jsx:94]: yes sir

Comment (saiky76) [reply bigbluebutton-html5/imports/ui/components/chat/component.jsx:94]: Then this should not appear in the final change, if you reverted all the old ones. I should see only the other file change. Looks like you added this new now. Without this change are things not working?

Comment (sravanikarri) [reply bigbluebutton-html5/imports/ui/components/chat/component.jsx:94]: this change was done long ago  pushed to chatupload when private chat is enabled  , thats why its showing as change sir

**Pull request merged**